recipes/shenfun/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set build = 0 %}
+{% set name = "shenfun" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/shenfun/archive/{{ version }}.tar.gz
+  sha256: 48e51882b2d45359997aee4de3d7dc46c81db2ee46b5ddfb330ebdb81ba312ad
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - numpy 1.16.*
```

Comment:
Do you really need numpy 1.16 for building? That seems rather high: https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
Well, at first I had nothing there, that failed. Then I tried 1.11.* and it failed as well. I get an error when a different numpy is used to build than to run, so I went with the latest numpy because that is what you're running with. See [this build](https://travis-ci.org/spectralDNS/shenfun/jobs/486460183) as an example of what goes wrong (could't find the staged build). It is using numpy 1.11 to build and numpy 1.15 in the test. It fails with `E   ValueError: numpy.ufunc size changed, may indicate binary incompatibility. Expected 216 from C header, got 192 from PyObject`. Works for python2.7 and 3.6, but fails for 3.7. Changing it such that the same numpy is used in the build and run and everything works. This is probably because I'm using cython to compile a few functions. I'm ok with hosting at 1.16.

Reply:
You mean the mpi4py-fft package should remove [this numpy pin](https://github.com/conda-forge/mpi4py-fft-feedstock/blob/master/recipe/conda_build_config.yaml)?

Reply:
I'm trying to rebuild now after stripping away scipy, mpi and mpi4py-fft from host, and setting numpy to 1.11.*. Lets see how it goes.

Reply:
Bumped the build number by mistake:-( Seemed to build just fine on circleci without scipy and mpi. I have just opened a [PR](https://github.com/conda-forge/mpi4py-fft-feedstock/pull/5) over at mpi4py-fft.